### DISCUSSION TITLE
Skip every third Occurence (RT)

### DISCUSSION THREAD

**Discussion by Roxas2712** [firefly-iii/firefly-iii#6273] [Firefly III]
Hi everyone,

i would like to skip every third occurence of a reoccuring transaction.
Therefore I created it and entered ‚3‘ in ‚Skip‘. What now happening is that the transaction is happening every third month instead. Does anybody know how I can archive my goal?

Thank you in advance :)

**Comment by JC5** [comment]
You can't, unless you create two recurring transactions. Set the, both to skip '3', and let them alternate.
